src/App.js

```diff
@@ -6,21 +7,34 @@ import TodoListItem from './todoListItem';
 
 
 
+
 const title = "Todo List";
 
+function useSemiPersistentState () {
+  // First you're getting the "savedTodoList" data from localStorage.
+  // Then you're parsing that data (which is a JSON string)
+  // Then you're passing the parsed data as the first argument to `useState`, which gives the state its default value (what `todoList` starts as)
+  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem("savedTodoList")));
```

Comment:
you'll need a || [] after the json.parse otherwise it'll throw an error on load if you don't have a savedTodoList object already saved to your local storage.